lib/cli/scripts/changelog.ts

```diff
@@ -113,18 +111,21 @@ function getCommits(options: DiffOptions): Array<Commit> {
     // https://stackoverflow.com/a/13928240/14644447
     log = log.trim().replace(/"/gm, '\\"').replace(/\^@\^/gm, '"');
     if (log.endsWith(',')) {
-        log = log.substring (0, log.length - 1);
+        log = log.substring(0, log.length - 1);
     }
 
-    return log.split('\n').map(str => JSON.parse(str) as Commit);
+    return log.split('\n').map(str => JSON.parse(str) as Commit).filter(commit => {
+        const filterRegex = RegExp(authorFilter);
```

Comment:
this can be moved out of the "filter" loop probably

Reply:
Extracted to a function